ReactApp/src/misc/api-fetch-setup.js

```diff
@@ -83,24 +80,6 @@ export function initializeApiFetch() {
 					template_lock: false,
 				},
 			},
-			[`/wp/v2/posts/${LOCAL_POST_ID}?context=edit`]: {
-				body: {
-					id: LOCAL_POST_ID,
-					slug: '',
-					status: 'auto-draft',
-					type: 'post',
-					title: { raw: 'Auto Draft', rendered: 'Auto Draft' },
-					content: {
-						raw: '',
-						rendered: '',
-						protected: false,
-						block_version: 0,
-					},
-					excerpt: { raw: '', rendered: '', protected: false },
-					author: LOCAL_AUTHOR_ID,
-					featured_media: 0,
-				},
-			},
```

Comment:
This was not needed anymore and it was causing problems when connecting with the API